vms/platformvm/txs/builder/backend.go

```diff
@@ -26,19 +25,16 @@ var (
 )
 
 func NewBackend(
-	cfg *config.Config,
 	state state.State,
 	atomicUTXOsMan avax.AtomicUTXOManager,
 ) *Backend {
 	return &Backend{
-		cfg:            cfg,
```

Comment:
I will need this for dynamic fees later on. I think I left it already to reduce diffs. It's not a big problem if we want to remove it here and re-add it later on